CHANGES

```diff
@@ -134,6 +134,18 @@ coqdep
   This is used by coq_makefile when generating dependencies for .v
   files (but not other files).
 
+Coq binaries and process model
+
+- Before 8.8, Coq distributed a single `coqtop` binary and a set of
```

Comment:
I guess this is now 8.9 and that this file conflicts

Reply:
Updated thx.